exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -0,0 +1,98 @@
+import io.grpc.ManagedChannel;
+import io.opentelemetry.exporters.jaeger.proto.api_v2.Collector;
+import io.opentelemetry.exporters.jaeger.proto.api_v2.CollectorServiceGrpc;
+import io.opentelemetry.exporters.jaeger.proto.api_v2.Model;
+import io.opentelemetry.proto.trace.v1.Span;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+
+/** Exports spans to Jaeger via gRPC, using Jaeger's protobuf model. */
+public class JaegerGrpcSpanExporter implements SpanExporter {
+  private final CollectorServiceGrpc.CollectorServiceBlockingStub blockingStub;
+  private final Model.Process process;
+
+  /**
+   * Creates a new Jaeger gRPC Span Reporter with the given name, using the given channel.
+   *
+   * @param serviceName this service's name
+   * @param channel the channel to use when communicating with the Jaeger Collector
+   */
+  public JaegerGrpcSpanExporter(String serviceName, ManagedChannel channel) {
+    String hostname;
+    String ipv4;
+
+    if (serviceName == null || serviceName.trim().length() == 0) {
+      throw new IllegalArgumentException("Service name must not be null or empty");
+    }
+
+    try {
+      hostname = InetAddress.getLocalHost().getHostName();
+      ipv4 = InetAddress.getLocalHost().getHostAddress();
+    } catch (UnknownHostException e) {
+      hostname = "(unknown)";
+      ipv4 = "0.0.0.0";
+    }
+
+    Model.KeyValue clientTag =
+        Model.KeyValue.newBuilder().setKey("jaeger.version").setVStr("opentelemetry-java").build();
+
+    Model.KeyValue ipv4Tag = Model.KeyValue.newBuilder().setKey("ip").setVStr(ipv4).build();
+
+    Model.KeyValue hostnameTag =
+        Model.KeyValue.newBuilder().setKey("hostname").setVStr(hostname).build();
+
+    this.process =
+        Model.Process.newBuilder()
+            .setServiceName(serviceName)
+            .addTags(clientTag)
+            .addTags(ipv4Tag)
+            .addTags(hostnameTag)
+            .build();
+    this.blockingStub = CollectorServiceGrpc.newBlockingStub(channel);
+  }
+
+  @Override
+  public ResultCode export(List<Span> spans) {
+    Model.Batch.Builder builder = Model.Batch.newBuilder();
+    builder.addAllSpans(Adapter.toJaeger(spans));
+    builder.setProcess(this.process);
+
+    Collector.PostSpansRequest.Builder requestBuilder = Collector.PostSpansRequest.newBuilder();
+    requestBuilder.setBatch(builder.build());
+    Collector.PostSpansRequest request = requestBuilder.build();
+
+    try {
+      // for now, there's nothing to check in the response object
+      //noinspection ResultOfMethodCallIgnored
+      this.blockingStub.postSpans(request);
+      return ResultCode.SUCCESS;
+    } catch (Throwable t) {
+      // TODO(jpkroehling) what are the possibilities here?
+      //  gRPC should will retry when the problem is in the connection
+      return ResultCode.FAILED_NONE_RETRYABLE;
+    }
+  }
+
+  @Override
+  public void shutdown() {}
```

Comment:
`channel.shutdown().awaitTermination(5, TimeUnit.SECONDS);` or any timeout you like for the moment. See https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-exporter.md#shutdown